{"name": "Тест неуспешной авторизации", "status": "passed", "steps": [{"name": "Проверка неуспешной авторизации", "status": "passed", "start": 1736988140031, "stop": 1736988141298}], "parameters": [{"name": "test_data", "value": "{'username': 'admin1', 'password': 'admin1', 'expected': 'Invalid credentials'}"}], "start": 1736988140031, "stop": 1736988141298, "uuid": "d688fb9d-24dc-4c2e-bbda-1e7cea94a2e8", "historyId": "0447667e0caf7111a6dc72c9bb0131d2", "testCaseId": "2b4f0dc9b560ddb4c4fbd568e058833b", "fullName": "tests.test_login_page.TestLogin#test_invalidLogin", "labels": [{"name": "feature", "value": "Тесты для авторизации"}, {"name": "story", "value": "Проверка неуспешной авторизации"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_page"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-M99MFDK"}, {"name": "thread", "value": "224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_page"}]}